binaries/build.py

```diff
@@ -1,44 +1,78 @@
 import argparse
+import glob
 import os
 import sys
-import glob
 
 # To help discover local modules
 REPO_ROOT = os.path.join(os.path.dirname(os.path.abspath(__file__)), "..")
 sys.path.append(REPO_ROOT)
 
-from pathlib import Path
 
-from ts_scripts.utils import is_conda_env, is_conda_build_env
-from binaries.conda.build_packages import conda_build, install_miniconda, install_conda_build
+from binaries.conda.build_packages import (
+    conda_build,
+    install_conda_build,
+    install_miniconda,
+)
+from ts_scripts.utils import is_conda_build_env, is_conda_env
 
 
-def build():
-    
-    print("## Started torchserve, model-archiver and workflow-archiver build")
-    create_wheel_cmd = "python setup.py bdist_wheel --release --universal"
+def build_dist_whl(nightly=False):
```

Comment:
Nit: Favor writing 2 separate functions instead of a function with a binary switch

Reply:
passing args instead for future extra args